src/elements/TextInput.tsx

```diff
@@ -12,10 +12,11 @@ import theme from "theme";
 import Typography from "./Typography";
 
 export interface TextInputProps extends InputHTMLAttributes<HTMLInputElement> {
-  readonly label?: string;
+  readonly endAdornment?: JSX.Element;
   readonly errorMessage?: string;
+  readonly label?: string;
   readonly startAdornment?: JSX.Element;
-  readonly endAdornment?: JSX.Element;
+  readonly widthType?: "default" | "full";
```

Comment:
Talked to design, they wanted to have 2 options for the text fields... either full width 100% or max width of `340px` to stay consistent throughout the app